packages/monday-api-mcp/src/utils/args/args.config.ts

```diff
@@ -29,4 +29,11 @@ export const ARG_CONFIGS: ArgConfig[] = [
     required: false,
     defaultValue: false,
   },
+  {
+    name: 'enableMondayAppsTools',
+    flags: ['--enable-monday-apps-tools', '-emat'],
```

Comment:
Let's change it to mode, so it'll be only apps and tools instead of addition of apps tools